ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -378,8 +378,10 @@ let rec cmdtable_data : (string * cmd_spec) list =
   ; ( "pool-certificate-uninstall"
     , {
         reqd= ["name"]
-      ; optn= []
-      ; help= "Uninstall a pool-wide TLS CA certificate."
+      ; optn= ["force"]
+      ; help= "Uninstall a pool-wide TLS CA certificate. The optional parameter \
+               '--force' will remove the DB entry even if the certificate file is \
+               non-existent"
       ; implementation= No_fd Cli_operations.pool_uninstall_ca_certificate
       ; flags= [Deprecated ["Use pool-uninstall-ca-certificate"]]
```

Comment:
What does this Deprecated flag actually do? It doesn't show when you do xe help pool-certificate-uninstall and it didn't show when I ran the command itself.

Reply:
This is probably because there's very little autogenerated code. The cli server should let the client print a message, probably. (will it break the internal regression tests, though?)